iwyu_include_picker.cc

```diff
@@ -1303,15 +1303,15 @@ vector<string> IncludePicker::GetCandidateHeadersForFilepath(
 vector<string> IncludePicker::GetCandidateHeadersForFilepathIncludedFrom(
     const string& included_filepath, const string& including_filepath) const {
   vector<string> retval;
-  const string quoted_includer = ConvertToQuotedInclude(including_filepath);
-  const string quoted_includee = ConvertToQuotedInclude(included_filepath);
+  const string quoted_includer = ConvertToQuotedInclude(including_filepath, MakeAbsolutePath(GetParentPath(including_filepath)));
```

Comment:
Does it work if you leave out the `including_filepath` argument? If so, I'd prefer that.